controllers/opentelemetrycollector_controller.go

```diff
@@ -75,6 +76,25 @@ type Params struct {
 	Config   config.Config
 }
 
+func (r *OpenTelemetryCollectorReconciler) Validate(otelcol v1beta1.OpenTelemetryCollector) admission.Warnings {
+	var warnings admission.Warnings
+	cfg := config.New(
+		config.WithCollectorImage("default-collector"),
+		config.WithTargetAllocatorImage("default-ta-allocator"),
+	)
+	params := manifests.Params{
```

Comment:
you should just use this method https://github.com/open-telemetry/opentelemetry-operator/blob/65477b55214a53495343e1d2a7e39c85b3c33290/controllers/opentelemetrycollector_controller.go#L191-L208